src/configure/mod.rs

```diff
@@ -574,7 +574,7 @@ pub enum Error {
     #[error("Invalid email address: {0:?}")]
     InvalidEmailAddress(String),
 
-    #[error("XML error at position {position}")]
+    #[error("XML error at position {position}: {error}")]
```

Comment:
wasn't your PR wrt the FFI error reporting supposed to automatically print causes?  Somehow manually chaining error descriptions like this seems a bit off, the `anyhow::Error` reporting-type with `{:#}` formatting should be taking care of this should it not?  My guess is that the actual problem is that whoever reports/logs this doesn't report the causes.

Reply:
Problem is, that's a `thiserror` error and thiserror does not do this: https://github.com/dtolnay/thiserror/issues/98